src/config/config.json

```diff
@@ -28,5 +28,11 @@
       "clientId": "MUST BE SET IN BABAJKA SECRET CONFIG",
       "clientSecret": "MUST BE SET IN BABAJKA SECRET CONFIG"
     }
+  },
+  "mail": {
```

Comment:
Please remember to update both `secret-dev.json` and `secret-staging.json` on our G Drive!

Reply:
I changed structure in https://github.com/babajka/babajka-backend/pull/47

please add it as `services: { mailchimp: { ... } }`